src/Resources/views/CRUD/base_array_macro.html.twig

```diff
@@ -8,17 +8,42 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 
 #}
-{% macro render_array(value, inline) %}
-    {% from _self import render_array %}
-    {% for key, val in value %}
-        {% if val is iterable %}
-            [{{ key }} => {{ render_array(val, inline) }}]
-        {% else %}
-            [{{ key }} => {{ val }}]
-        {% endif %}
+{% macro render_array(value, inline, values_only, translate, translation_domain, translation_prefix, translate_key) %}
```

Comment:
This is a BC-break if you don't add default values for the new-parameters.

Reply:
We could use `render_array(value, inline, options = [])` and maybe `render_array(value, options = [])` in the next major

Reply:
All new values are equal to false by default and I don't think that it's BC.
Group new params as `options` it's good idea, thanks!

Reply:
I may be wrong. I don't really know the behaviour in twig.

What if someone used the `render_array` macro in his own twig template ? 
If I still use `render_array(value, inline)` does I get an error ?

Reply:
Undefined params become `null`. But I like your idea of `options` argument and have implemented it.